docs/community/mxnet_channels.md

```diff
@@ -0,0 +1,6 @@
+# Join MXNet Development Discussion
+
+You can join the MXNet development discussion
```

Comment:
Cut this line because it's repetitive: "join the MXNet discussion (heading)... join the MXNet discussion (text)"